packages/destination-actions/src/destinations/google-enhanced-conversions/__tests__/google-enhanced-conversions.test.ts

```diff
@@ -203,5 +203,52 @@ describe('GoogleEnhancedConversions', () => {
       expect(responses.length).toBe(1)
       expect(responses[0].status).toBe(201)
     })
+
+    it('should rethrow the original error if error object does not contain error_statuses key', async () => {
+      const event = createTestEvent({
+        timestamp,
+        event: 'Test Event',
+        properties: {
+          email: 'janedoe@gmail.com',
+          orderId: '123',
+          firstName: 'Bob John',
+          lastName: 'Smith',
+          phone: '14150000000',
+          address: {
+            street: '123 Market Street',
+            city: 'San Francisco',
+            state: 'CA',
+            postalCode: '94000',
+            country: 'USA'
+          }
+        }
+      })
+
+      const error = new HTTPError(
+        {
+          url: 'https://www.google.com/ads/event/api/v1?conversion_tracking_id=699373583',
+          status: 400,
+          statusText: 'Unauthorized',
+          headers: []
+        },
+        {},
+        {}
+      )
+
+      nock('https://www.google.com/ads/event/api/v1')
+        .post(`?conversion_tracking_id=${conversionTrackingId}`)
+        .reply(400, error)
```

Comment:
I am not sure but i think just this should be enough to trigger the flow required. 
```suggestion
        .reply(400, {})
```